### ISSUE TITLE
File size not working properly

### ISSUE THREAD

**Issue by fredlkrue** [staxrip/staxrip#779]
**Describe the bug**
When setting an encode to a target file size the end up encode file is way off the target size.

**Expected behavior**
When using old version (StaxRip-2.1.8.0) the target size is only 1MB off but using the latest version (StaxRip-v2.8.0-x64) is off by many MBs.

**How to reproduce the issue**
Use latest version, set HEVC/x265 and use 2 pass encoding and set a target size like 700MB (make sure you enable the setting: "Use binary prefix (MiB) instead of decimal prefix (MB) for sizes)

Example:
I set up the target size to 700MB (please check the bitrate)
https://i.imgur.com/1IPjs7q.png
When setting up a target size of 701MB (check the bitrate)
https://i.imgur.com/UsyZtuu.png
And then when I change the bitrate between 701MB and 700MB check the target size (it goes below 700MB target size which shows a bug in the new target size calculation, this doesn't happen with version StaxRip-2.1.8.0 and below)
https://i.imgur.com/cWcsqZY.png

P.S.: and it doesn't even take into account the size of the sub file size (for example .sub file(s), which adds a few more MBs)

**Basically it should revert the changes made to target size since version 2.1.8.0**

**Comment by Dendraspis** [comment]
This has something to do with rounding and some minor bugs that existed in the old versions. 

But I can't reproduce your observation. My calculated values are always only 1MB off, mostly because of rounding issues:
![image](https://user-images.githubusercontent.com/50659978/135873226-143b8fc8-c3b8-49d3-a35e-c6cf4a1cc42b.png)
![image](https://user-images.githubusercontent.com/50659978/135873283-a96b46b1-c9a5-44ab-94a2-aa20a6117d08.png)
![image](https://user-images.githubusercontent.com/50659978/135873411-a4a95f35-bfda-49d6-95d8-60f811f077e3.png)
![image](https://user-images.githubusercontent.com/50659978/135873846-7f52e386-6e15-4a21-a871-47a1dca71e88.png)

> P.S.: and it doesn't even take into account the size of the sub file size (for example .sub file(s), which adds a few more MBs)

Sibtitles are taken into account, but because of compression in containers, it is hard to calculate the right size. As of v2.8.2 StaxRip takes the file size(s) divided by 3.

If anyone can reproduce this issue, we can try to re-investigate....
